docs/CHANGELOG.asciidoc

```diff
@@ -10,10 +10,496 @@
+====
+
+[discrete]
+[[breaking-113367]]
+.Removes `--plugin-dir` cli option
+[%collapsible]
+====
+*Details* +
+The `plugins.scanDirs` setting and `--plugin-dir` cli option have been removed. For more information, refer to {kibana-pull}113367[#113367].
+
+*Impact* +
+Before you upgrade to 8.0.0, remove `plugins.scanDirs` from kibana.yml.
+====
+
+[discrete]
+[[breaking-113296]]
+.Removes support for `optimize.*` settings
+[%collapsible]
+====
+*Details* +
+The legacy `optimize.*` settings have been removed. If your configuration uses the following legacy `optimize.*` settings, {kib} fails to start:
+
+* `optimize.lazy`
+* `optimize.lazyPort`
+* `optimize.lazyHost`
+* `optimize.lazyPrebuild`
+* `optimize.lazyProxyTimeout`
+* `optimize.enabled`
+* `optimize.bundleFilter`
+* `optimize.bundleDir`
+* `optimize.viewCaching`
+* `optimize.watch`
+* `optimize.watchPort`
+* `optimize.watchHost`
+* `optimize.watchPrebuild`
+* `optimize.watchProxyTimeout`
+* `optimize.useBundleCache`
+* `optimize.sourceMaps`
+* `optimize.workers`
+* `optimize.profile`
+* `optimize.validateSyntaxOfNodeModules`
+
+For more information, refer to {kibana-pull}113296[#113296].
+
+*Impact* +
+To run the `@kbn/optimizer` separately in development, pass `--no-optimizer` to `yarn start`. For more details, refer to {kibana-pull}73154[#73154]. 
+====
+
+[discrete]
+[[breaking-113173]]
+.Removes `so/server/es` settings
+[%collapsible]
+====
+*Details* +
+The `so/server/es` settings have been removed. If your configuration uses the following settings, {kib} fails to start:
+
+* `savedObjects.indexCheckTimeout`
+* `server.xsrf.token`
+* `elasticsearch.preserveHost`
+* `elasticsearch.startupTimeout`
+
+For more information, refer to {kibana-pull}113173[#113173].
+
+*Impact* +
+Before you upgrade to 8.0.0., remove these settings from kibana.yml.
+====
+
+[discrete]
+[[breaking-113068]]
+.Adds requirement for inline scripting
+[%collapsible]
+====
+*Details* +
+To start {kib}, you must enable inline scripting in {es}. For more information, refer to {kibana-pull}113068[#113068].
+
+*Impact* +
+Enable link:https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-security.html[inline scripting].
+====
+
+[discrete]
+[[breaking-112773]]
+.Removes `kibana.index` settings
+[%collapsible]
+====
+*Details* +
+The `kibana.index`, `xpack.reporting.index`, and `xpack.task_manager.index` settings have been removed. For more information, refer to {kibana-pull}112773[#112773].
+
+*Impact* +
+Use spaces, cross-cluster replication, or cross-cluster search. To migrate to <<xpack-spaces,spaces>>, export your <<managing-saved-objects, saved objects>> from a tenant into the default tenant in a space. For more details, refer to link:https://github.com/elastic/kibana/issues/82020[#82020].
```

Comment:
I'm not sure I understand the phrasing "default tenant in a space" -- I'd recommend removing it

```suggestion
Use spaces, cross-cluster replication, or cross-cluster search. To migrate to <<xpack-spaces,spaces>>, export your <<managing-saved-objects, saved objects>> from a tenant into the default space. For more details, refer to link:https://github.com/elastic/kibana/issues/82020[#82020].
```